spring-data-cassandra/src/test/java/org/springframework/data/cassandra/util/SpelUtilsTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2016-2017 the original author or authors.
```

Comment:
in this case, it's just `2017` (see the [Contribution guide](https://github.com/spring-projects/spring-data-build/blob/master/CONTRIBUTING.adoc) for more details).